cypress/e2e/issue-list.cy.ts

```diff
@@ -79,7 +97,6 @@ describe("Issue List", () => {
 
       cy.reload();
       cy.wait(["@getProjects", "@getIssuesPage2"]);
-      cy.wait(1500);
```

Comment:
Have you found this test to be flaky (meaning it randomly fails sometimes)? That's why I introduced this delay. I think the root cause is that the next router doesn't return the page query param during the first render. But it's always good to remove delays if not necessary.